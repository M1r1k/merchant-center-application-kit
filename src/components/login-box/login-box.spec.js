import React from 'react';
import { shallow } from 'enzyme';
import LoginBox from './login-box';

describe('rendering', () => {
  let wrapper;
  beforeEach(() => {
    const Foo = () => <div />;
    wrapper = shallow(
      <LoginBox>
        <Foo />
      </LoginBox>
    );
  });
  it('should output correct tree', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
