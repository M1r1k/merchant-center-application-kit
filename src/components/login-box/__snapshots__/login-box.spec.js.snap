// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should output correct tree 1`] = `
<Card
  className="box"
  theme="light"
  type="raised"
>
  <div
    className="header"
  >
    <div
      className="logo"
    >
      <img
        src="test-file-stub"
      />
    </div>
  </div>
  <div
    className="holder"
    data-test="login-form"
  >
    <Foo />
  </div>
</Card>
`;
